api/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java

```diff
@@ -150,6 +150,22 @@ public final class NessieConfigConstants {
   public static final String CONF_NESSIE_OAUTH2_CLIENT_SECRET =
       "nessie.authentication.oauth2.client-secret";
 
+  /**
+   * Extra parameters to include in each request to the token endpoint. This is useful for custom
+   * parameters that are not covered by the standard OAuth2.0 specification. Optional, defaults to
+   * empty.
+   *
+   * <p>The format of this field is a comma-separated list of key-value pairs, separated by an equal
+   * sign. The values must NOT be URL-encoded. Example:
+   *
+   * <pre>{@code
+   * nessie.authentication.oauth2.extra-params = "custom_param1=custom_value1,custom_param2=custom_value2"
```

Comment:
Should there be an example for the `audience` field, mentioning Auth0?

Reply:
Maybe an Auth0 guide in the site docs (not in javadoc)?